src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -85,23 +86,16 @@ function getChartConfig(props) {
   // also be passed in explicitly (e.g. for charts where there are more than one indicator)
   // so here we take the passed in value, or the value from the first indicator otherwise.
   const firstIndicator = INDICATORS[props.indicatorIds[0]];
-  const shortDescription = props.shortDescription || firstIndicator.shortDescription;
   const data = sortChartData(props.cities, props.indicatorIds[0]);
   const plotBands = getPlotBands(data, props.city, colorLight);
   const series = getSeries(props);
 
-  // We only want to show the short description as the chart title
-  // if the chart is not stacked
-  const yAxisTitle = isMultiple ? {} : { text: shortDescription };
```

Comment:
I've done some aligning of all the top elements (title, "last updated" legend and axis label). This includes always showing the axis label up the top, and never as part of the high chart.